Source/ACE/Entity/Player.cs

```diff
@@ -456,6 +478,42 @@ public void Load(AceCharacter character)
             Session.Network.EnqueueSend(setTurbineChatChannels, general, trade, lfg, roleplay);
         }
 
+        public AceObject GetSavableCharacter()
+        {
```

Comment:
Ok, but can of worms.  How do we save character-only data?  Are those all atomically separate?